File: Strategy/CSharp/Strategy/Behaivours/FlyWithRocket.cs
using System;

namespace Strategy.Behaivours
{
	public class FlyWithRocket : IFlyBehaviour
	{
		public FlyWithRocket ()
		{
		}
		
		public void fly()
		{
			Console.WriteLine("Flying with a rocket");
		}
		
	}
}

File: PHP/Strategy/Behaviours/FlyWithRocket.php
<?php

/**
 * FlyWithRocket.php
 *
 * Implements a strategy
 * @author Indika Piyasena
 * @version 1.0
 */

require_once 'IFlyBehaviour.php';

class FlyWithRocket implements IFlyBehaviour
{

	/**
	 * A way of flying
	 * @return void
	 */
	function fly()
	{
		Print "Flying with a rocket\n";
	}
}

?>